SPECS/kernel-alt.spec

```diff
@@ -55,652 +55,642 @@ Requires(posttrans): xcp-python-libs >= 3.0.2-4.2.xcpng8.3
+#Patch209: 0001-gfs2-Issue-revokes-more-intelligently.patch
+#Patch210: 0001-gfs2-Prepare-to-withdraw-as-soon-as-an-IO-error-occu.patch
+#Patch211: 0001-gfs2-Check-for-log-write-errors-before-telling-dlm-t.patch
+#Patch212: 0001-gfs2-Do-log_flush-in-gfs2_ail_empty_gl-even-if-ail-l.patch
+#Patch213: 0001-gfs2-Withdraw-in-gfs2_ail1_flush-if-write_cache_page.patch
+#Patch214: 0001-gfs2-drain-the-ail2-list-after-io-errors.patch
+#Patch215: 0001-gfs2-Don-t-demote-a-glock-until-its-revokes-are-writ.patch
+#Patch216: 0001-gfs2-Do-proper-error-checking-for-go_sync-family-of-.patch
+#Patch217: 0001-gfs2-flesh-out-delayed-withdraw-for-gfs2_log_flush.patch
+#Patch218: 0001-gfs2-don-t-allow-releasepage-to-free-bd-still-used-f.patch
+#Patch219: 0001-gfs2-allow-journal-replay-to-hold-sd_log_flush_lock.patch
+#Patch220: 0001-gfs2-leaf_dealloc-needs-to-allocate-one-more-revoke.patch
+#Patch221: 0001-gfs2-Additional-information-when-gfs2_ail1_flush-wit.patch
+#Patch222: 0001-gfs2-Clean-up-inode-initialization-and-teardown.patch
+#Patch223: 0001-gfs2-Switch-to-list_-first-last-_entry.patch
+#Patch224: 0001-gfs2-eliminate-gfs2_rsqa_alloc-in-favor-of-gfs2_qa_a.patch
+#Patch225: 0001-gfs2-Change-inode-qa_data-to-allow-multiple-users.patch
+#Patch226: 0001-gfs2-Split-gfs2_rsqa_delete-into-gfs2_rs_delete-and-.patch
+#Patch227: 0001-gfs2-Remove-unnecessary-gfs2_qa_-get-put-pairs.patch
+#Patch228: 0001-gfs2-don-t-lock-sd_log_flush_lock-in-try_rgrp_unlink.patch
+#Patch229: 0001-gfs2-instrumentation-wrt-ail1-stuck.patch
+#Patch230: 0001-gfs2-change-from-write-to-read-lock-for-sd_log_flush.patch
+#Patch231: 0001-gfs2-Fix-oversight-in-gfs2_ail1_flush.patch
+#Patch232: 0001-dlm-Switch-to-using-wait_event.patch
+#Patch233: 0001-dlm-use-the-tcp-version-of-accept_from_sock-for-sctp.patch
+Patch234: 0002-net-add-sock_set_reuseaddr.patch
+Patch235: 0003-net-add-sock_set_sndtimeo.patch
+Patch236: 0004-net-add-sock_set_keepalive.patch
+Patch237: 0005-net-add-sock_set_rcvbuf.patch
+Patch238: 0006-tcp-add-tcp_sock_set_nodelay.patch
+Patch239: 0007-sctp-add-sctp_sock_set_nodelay.patch
+# GVFS2/DLM not supported
+#Patch240: 0009-dlm-dlm_internal-Replace-zero-length-array-with-flex.patch
+#Patch241: 0010-dlm-user-Replace-zero-length-array-with-flexible-arr.patch
+#Patch242: 0011-fs-dlm-remove-unneeded-semicolon-in-rcom.c.patch
+#Patch243: 0012-dlm-remove-BUG-before-panic.patch
+#Patch244: 0001-gfs2-fix-withdraw-sequence-deadlock.patch
+#Patch245: 0001-gfs2-Fix-error-exit-in-do_xmote.patch
+#Patch246: 0001-gfs2-Fix-BUG-during-unmount-after-file-system-withdr.patch
+#Patch247: 0001-gfs2-Fix-use-after-free-in-gfs2_logd-after-withdraw.patch
+#Patch248: 0001-block-call-rq_qos_exit-after-queue-is-frozen.patch
+Patch249: 0001-scsi-libfc-free-response-frame-from-GPN_ID.patch
+#Patch250: 0001-xen-xenbus-ensure-xenbus_map_ring_valloc-returns-pro.patch
+#Patch251: 0013-treewide-Remove-uninitialized_var-usage.patch
+#Patch252: 0014-dlm-Fix-kobject-memleak.patch
```

Comment:
Merged upstream, DLM not supported, or both?

Reply:
Fixed